### PR TITLE
fdroidserver 2.0.1

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/fdroidserver.rb [hunk 1]
```diff
@@ -359,8 +362,9 @@ def install
     end
 
     venv.pip_install resource("cffi") # or bcrypt fails to build
+    venv.pip_install resource("wheel") # or kiwisolver fails to build
 
-    res = resources.map(&:name).to_set - %w[cffi lxml Pillow ptyprocess]
+    res = resources.map(&:name).to_set - %w[cffi lxml Pillow ptyprocess wheel]
```
Comment (Rylan12) [inline Formula/fdroidserver.rb:367]: Am I missing something or are `cffi` and `lxml` never installed?

**Review by Rylan12 — COMMENTED** [review]
`exclude_packages` use looks good to me. I'm glad a use for it finally showed up in homebrew/core so there's an example that uses it.

File: Formula/fdroidserver.rb (hunk 1 above)
Comment (Rylan12) [reply Formula/fdroidserver.rb:367]: ... 🤦 it's literally two lines above ignore this 😅

File: Formula/fdroidserver.rb [hunk 2]
```diff
@@ -339,9 +345,6 @@ def install
 
       # avoid triggering "helpful" distutils code that doesn't recognize Xcode 7 .tbd stubs
       ENV.delete "SDKROOT"
-      unless MacOS::CLT.installed?
-        ENV.append "CFLAGS", "-I#{MacOS.sdk_path}/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers"
-      end
```
Comment (Rylan12) [inline Formula/fdroidserver.rb:344]: Is this why the `tcl-tk` dependency was added?

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**